rpc/tests/share.rs

```diff
@@ -3,20 +3,22 @@ use celestia_types::consts::appconsts::{
     CONTINUATION_SPARSE_SHARE_CONTENT_SIZE, FIRST_SPARSE_SHARE_CONTENT_SIZE, SEQUENCE_LEN_BYTES,
     SHARE_INFO_BYTES,
 };
-use celestia_types::nmt::NamespacedHash;
+use celestia_types::nmt::Namespace;
 use celestia_types::Blob;
-use jsonrpsee::http_client::HttpClient;
 
 pub mod utils;
 
-use utils::{random_bytes, random_ns, test_client, AuthLevel};
+use crate::utils::client::blob_submit;
+use crate::utils::{random_bytes, random_ns, random_ns_range, test_client, AuthLevel};
 
-async fn test_get_shares_by_namespace(client: &HttpClient) {
+#[tokio::test]
+async fn test_get_shares_by_namespace() {
```

Comment:
could you remove all the `test_` prefixes? I added them because it was just a regular function before